<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: &quot;#09182a&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="toSettingImg">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>867</y>
      <width>56</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/icon_eqsetting.png);
background-color:  &quot;#00000000&quot;;
image: url(:/img/icon_eqsetting.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../tcpMusic/TCPMusicPlayerV1/img.qrc">
      <normaloff>:/img/icon_eqsetting.png</normaloff>:/img/icon_eqsetting.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="songImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="songName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>390</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 23pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="singerName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>55</y>
      <width>311</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="songDetail">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>311</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 18pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyHSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>500</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="tracking">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>155</y>
      <width>540</width>
      <height>660</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #0d2440;
selection-color:  #0d2440;
selection-background-color:  #0d2440;
</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="gridSize">
     <size>
      <width>540</width>
      <height>66</height>
     </size>
    </property>
   </widget>
   <widget class="MyHSlider" name="songProSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>816</y>
      <width>460</width>
      <height>36</height>
     </rect>
    </property>
    <property name="pageStep">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="position">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>816</y>
      <width>40</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="duraction">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>816</y>
      <width>40</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="scanning1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>没找到音乐.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="playImg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>860</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/icon_play.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextSongImg">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>860</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/icon_next_normal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playModeImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>870</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/icon_list.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="lastSongImg">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>860</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/icon_last_normal.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="switchList">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/icon_eqsetting.png);
background-color:  &quot;#00000000&quot;;
image: url(:/img/change.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../tcpMusic/TCPMusicPlayerV1/img.qrc">
      <normaloff>:/img/change.png</normaloff>:/img/change.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pageDown">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>714</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  &quot;#00000000&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../tcpMusic/TCPMusicPlayerV1/img.qrc">
      <normaloff>:/img/icon_down_normal.png</normaloff>
      <normalon>:/img/icon_down_selected.png</normalon>
      <activeoff>:/img/icon_down_normal.png</activeoff>
      <activeon>:/img/icon_down_selected.png</activeon>
      <selectedoff>:/img/icon_down_normal.png</selectedoff>
      <selectedon>:/img/icon_down_selected.png</selectedon>:/img/icon_down_normal.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pageUp">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>184</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  &quot;#00000000&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../tcpMusic/TCPMusicPlayerV1/img.qrc">
      <normaloff>:/img/icon_up_normal.png</normaloff>
      <normalon>:/img/icon_up_selected.png</normalon>
      <activeoff>:/img/icon_up_normal.png</activeoff>
      <activeon>:/img/icon_up_selected.png</activeon>
      <selectedoff>:/img/icon_up_normal.png</selectedoff>
      <selectedon>:/img/icon_up_selected.png</selectedon>:/img/icon_up_normal.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="errorDialog" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>540</width>
      <height>960</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: &quot;#2effffff&quot;;</string>
    </property>
    <widget class="QLabel" name="errorTips">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>430</y>
       <width>300</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(37, 38, 43);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>版权错误！你使用的可能是盗版软件！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </widget>
   <zorder>songDetail</zorder>
   <zorder>songImg</zorder>
   <zorder>songName</zorder>
   <zorder>singerName</zorder>
   <zorder>volumeSlider</zorder>
   <zorder>listWidget</zorder>
   <zorder>songProSlider</zorder>
   <zorder>position</zorder>
   <zorder>duraction</zorder>
   <zorder>scanning1</zorder>
   <zorder>playImg</zorder>
   <zorder>nextSongImg</zorder>
   <zorder>playModeImg</zorder>
   <zorder>lastSongImg</zorder>
   <zorder>toSettingImg</zorder>
   <zorder>switchList</zorder>
   <zorder>pageDown</zorder>
   <zorder>pageUp</zorder>
   <zorder>errorDialog</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyHSlider</class>
   <extends>QSlider</extends>
   <header>myhslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
